src/Fields/field.jl

```diff
@@ -781,3 +781,5 @@ function fill_halo_regions!(field::Field, args...; kwargs...)
     return nothing
 end
 
+# return a fields boundary conditions
+get_boundary_conditions(field::Field) = field.boundary_conditions
```

Comment:
There's already a function called `boundary_conditions` that does this

Reply:
Oh thanks, I had to add a place holder for this to the top level since boundary conditions are defined before fields